src/dmd/expression.d

```diff
@@ -4696,6 +4696,7 @@ extern (C++) final class CallExp : UnaExp
     Expressions* arguments; // function arguments
     FuncDeclaration f;      // symbol to call
     bool directcall;        // true if a virtual call is devirtualized
+    bool inDebugStatement;  // true if this was in a debug statement
```

Comment:
H...h...h...

Reply:
I presume you mean fix the corresponding .h file.